roles/openshift_registry/tasks/main.yml

```diff
@@ -1,11 +1,14 @@
 ---
-- set_fact: _oreg_images="--images={{ oreg_url|quote }}"
-  when: oreg_url is defined
+
+- set_fact: _oreg_images="--images='{{ openshift.master.registry_url }}'"
+
+- set_fact: _oreg_selector="--selector='{{ openshift.master.registry_selector }}'"
 
 - name: Deploy OpenShift Registry
   command: >
     {{ openshift.common.admin_binary }} registry
-    --create
+    --create --service-account=registry
+    {{ _oreg_selector|default() }} --replicas=0
     --credentials={{ openshift_master_config_dir }}/openshift-registry.kubeconfig {{ _oreg_images|default() }}
```

Comment:
I think we should drop the |default() on both the _oreg_selector and _oreg_images in lines 11 & 12 here because in no situation would they be null. They'll always be at least "-images=" and "--selector=" respectively.